workflows/quickies/phylogenetic-context/Q1-phylogenetic-context-analysis.ga

```diff
@@ -0,0 +1 @@
ele, and Alternative allele", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "errors": null, "id": 16, "input_connections": {"in_file": {"id": 15, "output_name": "out_file1"}}, "inputs": [], "label": null, "name": "Datamash", "outputs": [{"name": "out_file", "type": "tabular"}], "position": {"bottom": 924.546875, "height": 116, "left": 803.0625, "right": 1003.0625, "top": 808.546875, "width": 200, "x": 803.0625, "y": 808.546875}, "post_job_actions": {"HideDatasetActionout_file": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "out_file"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "tool_shed_repository": {"changeset_revision": "562f3c677828", "name": "datamash_ops", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"grouping\": \"3,5,6\", \"header_in\": \"true\", \"header_out\": \"false\", \"ignore_case\": \"false\", \"in_file\": {\"__class__\": \"ConnectedValue\"}, \"need_sort\": \"true\", \"operations\": [{\"__index__\": 0, \"op_name\": \"countunique\", \"op_column\": \"1\"}, {\"__index__\": 1, \"op_name\": \"unique\", \"op_column\": \"1\"}], \"print_full_line\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.1.0", "type": "tool", "uuid": "8ed33929-a75d-42fe-993a-db7be898683c", "workflow_outputs": []}, "17": {"annotation": "Here additional grouping summarizes types of changes that exist in this sample", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "errors": null, "id": 17, "input_connections": {"in_file": {"id": 16, "output_name": "out_file"}}, "inputs": [{"description": "runtime parameter for tool Datamash", "name": "in_file"}], "label": "Substitution matrix", "name": "Datamash", "outputs": [{"name": "out_file", "type": "tabular"}], "position": {"bottom": 868.546875, "height": 137, "left": 1078.046875, "right": 1278.046875, "top": 731.546875, "width": 200, "x": 1078.046875, "y": 731.546875}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "tool_shed_repository": {"changeset_revision": "562f3c677828", "name": "datamash_ops", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"grouping\": \"2,3\", \"header_in\": \"false\", \"header_out\": \"false\", \"ignore_case\": \"false\", \"in_file\": {\"__class__\": \"RuntimeValue\"}, \"need_sort\": \"true\", \"operations\": [{\"__index__\": 0, \"op_name\": \"count\", \"op_column\": \"1\"}], \"print_full_line\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.1.0", "type": "tool", "uuid": "d116b621-a706-4b31-ae4f-16466eeaeda1", "workflow_outputs": [{"label": "Substitution frequency matrix", "output_name": "out_file", "uuid": "b0d49893-ca7e-4779-856c-48eba92ac42d"}]}}, "tags": ["viral", "bacterial", "quickies", "variation", "phylogeny"], "uuid": "72a03e89-abd9-46d5-a7c6-b88717989e19", "version": 14}
```

Comment:
```suggestion
{"a_galaxy_workflow": "true", "annotation": "A workflow for quick comparison of viral or bacterial genomes against existing data (e.g., where my coronaviral genomes fit within all available data?)", "creator": [{"class": "Person", "identifier": "0000-0002-5987-8032", "name": "Anton Nekrutenko"}], "release": "0.1", "format-version": "0.1", "name": "Q1 phylogenetic context analysis", "steps": {"0": {"annotation": "These are sequences (genomes) you would like to compare to existing data and see where they fit in phylogenetically and in terms of sequence variation. They are uploaded as a COLLECTION.", "content_id": null, "errors": null, "id": 0, "input_connections": {}, "inputs": [{"description": "These are sequences (genomes) you would like to compare to existing data and see where they fit in phylogenetically and in terms of sequence variation. They are uploaded as a COLLECTION.", "name": "My sequences"}], "label": "My sequences", "name": "Input dataset collection", "outputs": [], "position": {"bottom": 88.5, "height": 63, "left": -1725, "right": -1525, "top": 25.5, "width": 200, "x": -1725, "y": 25.5}, "tool_id": null, "tool_state": "{\"optional\": false, \"tag\": \"\", \"collection_type\": \"list\"}", "tool_version": null, "type": "data_collection_input", "uuid": "052e3fea-2df7-492c-a3f9-0a1f61324201", "workflow_outputs": []}, "1": {"annotation": "This is a list of accession numbers for already existing genomes", "content_id": null, "errors": null, "id": 1, "input_connections": {}, "inputs": [{"description": "This is a list of accession numbers for already existing genomes", "name": "List of NCBI accessions"}], "label": "List of NCBI accessions", "name": "Input dataset", "outputs": [], "position": {"bottom": 213.53125, "height": 84, "left": -1720.984375, "right": -1520.984375, "top": 129.53125, "width": 200, "x": -1720.984375, "y": 129.53125}, "tool_id": null, "tool_state": "{\"optional\": false, \"tag\": \"\"}", "tool_version": null, "type": "data_input", "uuid": "6b509593-9e64-4737-b35f-68e8e6d7b020", "workflow_outputs": []}, "2": {"annotation": "This is a representative reference genome (in GenBank format)", "content_id": null, "errors": null, "id": 2, "input_connections": {}, "inputs": [{"description": "This is a representative reference genome (in GenBank format)", "name": "Reference"}], "label": "Reference", "name": "Input dataset", "outputs": [], "position": {"bottom": 330.515625, "height": 63, "left": -1724.453125, "right": -1524.453125, "top": 267.515625, "width": 200, "x": -1724.453125, "y": 267.515625}, "tool_id": null, "tool_state": "{\"optional\": false, \"format\": [\"genbank\"], \"tag\": \"\"}", "tool_version": null, "type": "data_input", "uuid": "547e4072-d8b2-44c8-bffa-48e3001e5fce", "workflow_outputs": []}, "3": {"annotation": "This step downloads sequences (in FASTA) format from NCBI to Galaxy", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/ncbi_acc_download/ncbi_acc_download/0.2.7+galaxy0", "errors": null, "id": 3, "input_connections": {"query_source|accession_file": {"id": 1, "output_name": "output"}}, "inputs": [], "label": "Data downloaded from NCBI", "name": "NCBI Accession Download", "outputs": [{"name": "output", "type": "input"}, {"name": "error_log", "type": "txt"}, {"name": "failed_accessions", "type": "txt"}], "position": {"bottom": 513.53125, "height": 346, "left": -1343.953125, "right": -1143.953125, "top": 167.53125, "width": 200, "x": -1343.953125, "y": 167.53125}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/ncbi_acc_download/ncbi_acc_download/0.2.7+galaxy0", "tool_shed_repository": {"changeset_revision": "28587613264f", "name": "ncbi_acc_download", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"ignore_failed\": \"1\", \"molecule\": {\"select\": \"nucleotide\", \"__current_case__\": 0, \"format\": \"fasta\"}, \"query_source\": {\"select\": \"accession_file\", \"__current_case__\": 0, \"accession_file\": {\"__class__\": \"ConnectedValue\"}}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "0.2.7+galaxy0", "type": "tool", "uuid": "bd7b6a6a-56d3-4f65-bd0c-f15ce49caadb", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "965f2899-60f7-41b9-b4ff-f5fcb4a4a3c5"}, {"label": null, "output_name": "error_log", "uuid": "83d09eeb-6601-4c28-a99a-aa6fbecb438b"}, {"label": null, "output_name": "failed_accessions", "uuid": "5d6e53b0-476f-4271-a909-ceec2a9ec47b"}]}, "4": {"annotation": "This step builds a SNPEff database. It will be used to annotate differences between genomes (which are Synonymous and which are now and so on).  ", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpeff/snpEff_build_gb/4.3+T.galaxy4", "errors": null, "id": 4, "input_connections": {"input_type|input_gbk": {"id": 2, "output_name": "output"}}, "inputs": [], "label": "FASTA sequence of the reference", "name": "SnpEff build:", "outputs": [{"name": "snpeff_output", "type": "snpeffdb"}, {"name": "output_fasta", "type": "fasta"}], "position": {"bottom": 1359.5625, "height": 210, "left": -1340.9375, "right": -1140.9375, "top": 1149.5625, "width": 200, "x": -1340.9375, "y": 1149.5625}, "post_job_actions": {"HideDatasetActionsnpeff_output": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "snpeff_output"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpeff/snpEff_build_gb/4.3+T.galaxy4", "tool_shed_repository": {"changeset_revision": "cfcf33df7fc0", "name": "snpeff", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"codon_table\": \"Standard\", \"genome_version\": \"reference\", \"input_type\": {\"input_type_selector\": \"gb\", \"__current_case__\": 0, \"input_gbk\": {\"__class__\": \"ConnectedValue\"}, \"fasta\": \"yes\", \"remove_version\": \"true\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "4.3+T.galaxy4", "type": "tool", "uuid": "b4383a01-0311-4c08-b5e2-48c27032de99", "workflow_outputs": [{"label": null, "output_name": "output_fasta", "uuid": "41c8cbc7-d964-4dd5-a206-8355ea227e1e"}]}, "5": {"annotation": "Here your sequences are merged with sequences downloaded from NCBI", "content_id": "__MERGE_COLLECTION__", "errors": null, "id": 5, "input_connections": {"inputs_0|input": {"id": 0, "output_name": "output"}, "inputs_1|input": {"id": 3, "output_name": "output"}}, "inputs": [], "label": null, "name": "Merge collections", "outputs": [{"name": "output", "type": "input"}], "position": {"bottom": 322.53125, "height": 210, "left": -965.953125, "right": -765.953125, "top": 112.53125, "width": 200, "x": -965.953125, "y": 112.53125}, "post_job_actions": {}, "tool_id": "__MERGE_COLLECTION__", "tool_state": "{\"advanced\": {\"conflict\": {\"duplicate_options\": \"keep_first\", \"__current_case__\": 3}}, \"inputs\": [{\"__index__\": 0, \"input\": {\"__class__\": \"ConnectedValue\"}}, {\"__index__\": 1, \"input\": {\"__class__\": \"ConnectedValue\"}}], \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.0.0", "type": "tool", "uuid": "b42d6c51-e911-41ad-8f98-3d6746132ae1", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "38dffd86-95c4-4518-8b47-511370149bf3"}]}, "6": {"annotation": "At this step every sequence from the merged collection (yours + existing ones from NCBI) is mapped against the reference using minimap2 using specific settings specified in \"Select a profile of a select options\"", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/minimap2/minimap2/2.24+galaxy0", "errors": null, "id": 6, "input_connections": {"fastq_input|fastq_input1": {"id": 5, "output_name": "output"}, "reference_source|ref_file": {"id": 4, "output_name": "output_fasta"}}, "inputs": [], "label": null, "name": "Map with minimap2", "outputs": [{"name": "alignment_output", "type": "bam"}], "position": {"bottom": 338.546875, "height": 231, "left": -658.953125, "right": -458.953125, "top": 107.546875, "width": 200, "x": -658.953125, "y": 107.546875}, "post_job_actions": {"HideDatasetActionalignment_output": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "alignment_output"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/minimap2/minimap2/2.24+galaxy0", "tool_shed_repository": {"changeset_revision": "11a0d50a54e6", "name": "minimap2", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"__job_resource\": {\"__job_resource__select\": \"no\", \"__current_case__\": 0}, \"alignment_options\": {\"splicing\": {\"splice_mode\": \"preset\", \"__current_case__\": 0}, \"A\": null, \"B\": null, \"O\": null, \"O2\": null, \"E\": null, \"E2\": null, \"z\": null, \"z2\": null, \"s\": null, \"no_end_flt\": \"true\"}, \"fastq_input\": {\"fastq_input_selector\": \"single\", \"__current_case__\": 0, \"fastq_input1\": {\"__class__\": \"ConnectedValue\"}, \"analysis_type_selector\": \"asm5\"}, \"indexing_options\": {\"H\": \"false\", \"k\": null, \"w\": null, \"I\": null}, \"io_options\": {\"output_format\": \"BAM\", \"Q\": \"false\", \"L\": \"false\", \"K\": null, \"cs\": null, \"c\": \"false\", \"eqx\": \"false\", \"Y\": \"false\"}, \"mapping_options\": {\"N\": null, \"F\": null, \"f\": null, \"kmer_ocurrence_interval\": {\"interval\": \"\", \"__current_case__\": 1}, \"min_occ_floor\": null, \"q_occ_frac\": \"0.01\", \"g\": null, \"r\": null, \"n\": null, \"m\": null, \"max_chain_skip\": null, \"max_chain_iter\": null, \"X\": \"false\", \"p\": null, \"mask_len\": null}, \"reference_source\": {\"reference_source_selector\": \"history\", \"__current_case__\": 1, \"ref_file\": {\"__class__\": \"ConnectedValue\"}}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "2.24+galaxy0", "type": "tool", "uuid": "59cdbf64-d5e7-4d5c-946a-404407f6340b", "workflow_outputs": []}, "7": {"annotation": "Mapped data is used to call variants using lofreq using the most relaxed settings. ", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/lofreq_call/lofreq_call/2.1.5+galaxy1", "errors": null, "id": 7, "input_connections": {"reads": {"id": 6, "output_name": "alignment_output"}, "reference_source|ref": {"id": 4, "output_name": "output_fasta"}}, "inputs": [], "label": null, "name": "Call variants", "outputs": [{"name": "variants", "type": "vcf"}], "position": {"bottom": 543.546875, "height": 126, "left": -642.9375, "right": -442.9375, "top": 417.546875, "width": 200, "x": -642.9375, "y": 417.546875}, "post_job_actions": {"HideDatasetActionvariants": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "variants"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/lofreq_call/lofreq_call/2.1.5+galaxy1", "tool_shed_repository": {"changeset_revision": "e1461b5c52a0", "name": "lofreq_call", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"__job_resource\": {\"__job_resource__select\": \"no\", \"__current_case__\": 0}, \"call_control\": {\"set_call_options\": \"yes\", \"__current_case__\": 1, \"coverage\": {\"min_cov\": \"1\", \"max_depth\": \"1000000\"}, \"pe\": {\"use_orphan\": \"false\"}, \"bc_quals\": {\"min_bq\": \"0\", \"min_alt_bq\": \"0\", \"alt_bq\": {\"modify\": \"custom\", \"__current_case__\": 2, \"def_alt_bq\": \"1\"}}, \"align_quals\": {\"alnqual\": {\"use_alnqual\": \"-A -B\", \"__current_case__\": 2}}, \"map_quals\": {\"min_mq\": \"0\", \"use_mq\": {\"no_mq\": \"\", \"__current_case__\": 0, \"max_mq\": \"255\"}}, \"source_qual\": {\"use_src_qual\": {\"src_qual\": \"\", \"__current_case__\": 0}}, \"joint_qual\": {\"min_jq\": \"0\", \"min_alt_jq\": \"0\", \"def_alt_jq\": \"0\"}}, \"filter_control\": {\"filter_type\": \"set_all_off\", \"__current_case__\": 0, \"sig\": \"1\", \"bonf\": \"1\", \"others\": \"--no-default-filter\"}, \"reads\": {\"__class__\": \"ConnectedValue\"}, \"reference_source\": {\"ref_selector\": \"history\", \"__current_case__\": 1, \"ref\": {\"__class__\": \"ConnectedValue\"}}, \"regions\": {\"restrict_to_region\": \"genome\", \"__current_case__\": 0}, \"variant_types\": \"--call-indels\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "2.1.5+galaxy1", "type": "tool", "uuid": "34e4e6ab-d1fa-4cf0-af84-5ad09224e89e", "workflow_outputs": []}, "8": {"annotation": "We use bcftools consensus to convert variants into consensus sequences", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/bcftools_consensus/bcftools_consensus/1.10+galaxy1", "errors": null, "id": 8, "input_connections": {"input_file": {"id": 7, "output_name": "variants"}, "reference_source|fasta_ref": {"id": 4, "output_name": "output_fasta"}}, "inputs": [{"description": "runtime parameter for tool bcftools consensus", "name": "sec_default"}], "label": null, "name": "bcftools consensus", "outputs": [{"name": "output_file", "type": "fasta"}], "position": {"bottom": 362.546875, "height": 220, "left": -286.9375, "right": -86.9375, "top": 142.546875, "width": 200, "x": -286.9375, "y": 142.546875}, "post_job_actions": {"HideDatasetActionoutput_file": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "output_file"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/bcftools_consensus/bcftools_consensus/1.10+galaxy1", "tool_shed_repository": {"changeset_revision": "92182c270ce4", "name": "bcftools_consensus", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"chain\": \"false\", \"input_file\": {\"__class__\": \"ConnectedValue\"}, \"reference_source\": {\"reference_source_selector\": \"history\", \"__current_case__\": 1, \"fasta_ref\": {\"__class__\": \"ConnectedValue\"}}, \"rename\": \"true\", \"sec_default\": {\"mask\": {\"__class__\": \"RuntimeValue\"}, \"iupac_codes\": \"false\", \"sample\": \"\", \"select_haplotype\": null}, \"sec_restrict\": {\"include\": \"\", \"exclude\": \"\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.10+galaxy1", "type": "tool", "uuid": "439637cb-bfec-4bf0-85ee-2fadf181deb0", "workflow_outputs": []}, "9": {"annotation": "Here the variants are annotated (coding, non-coding etc.)", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpeff/snpEff/4.3+T.galaxy1", "errors": null, "id": 9, "input_connections": {"input": {"id": 7, "output_name": "variants"}, "snpDb|snpeff_db": {"id": 4, "output_name": "snpeff_output"}}, "inputs": [{"description": "runtime parameter for tool SnpEff eff:", "name": "intervals"}, {"description": "runtime parameter for tool SnpEff eff:", "name": "transcripts"}], "label": null, "name": "SnpEff eff:", "outputs": [{"name": "snpeff_output", "type": "vcf"}], "position": {"bottom": 899.546875, "height": 251, "left": -374.9375, "right": -174.9375, "top": 648.546875, "width": 200, "x": -374.9375, "y": 648.546875}, "post_job_actions": {"HideDatasetActionsnpeff_output": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "snpeff_output"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpeff/snpEff/4.3+T.galaxy1", "tool_shed_repository": {"changeset_revision": "cfcf33df7fc0", "name": "snpeff", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"annotations\": [\"-formatEff\"], \"chr\": \"\", \"csvStats\": \"false\", \"filter\": {\"specificEffects\": \"no\", \"__current_case__\": 0}, \"filterOut\": [\"-no-downstream\", \"-no-intergenic\", \"-no-intron\", \"-no-upstream\", \"-no-utr\"], \"generate_stats\": \"false\", \"input\": {\"__class__\": \"ConnectedValue\"}, \"inputFormat\": \"vcf\", \"intervals\": {\"__class__\": \"RuntimeValue\"}, \"noLog\": \"true\", \"offset\": \"default\", \"outputConditional\": {\"outputFormat\": \"vcf\", \"__current_case__\": 0}, \"snpDb\": {\"genomeSrc\": \"custom\", \"__current_case__\": 3, \"snpeff_db\": {\"__class__\": \"ConnectedValue\"}, \"codon_table\": \"Standard\"}, \"spliceRegion\": {\"setSpliceRegions\": \"no\", \"__current_case__\": 0}, \"spliceSiteSize\": null, \"transcripts\": {\"__class__\": \"RuntimeValue\"}, \"udLength\": \"0\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "4.3+T.galaxy1", "type": "tool", "uuid": "54e95f18-f9b7-4130-a395-48297ba76834", "workflow_outputs": []}, "10": {"annotation": "Collection of consensus is collapsed into a multi-fasta file. Because we used the same reference sequence to create consenses this is effectively a multiple alignment that can be used, for example, for phylogenetic tree reconstruction.", "content_id": "toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0", "errors": null, "id": 10, "input_connections": {"input_list": {"id": 8, "output_name": "output_file"}}, "inputs": [], "label": "Multiple alignment", "name": "Collapse Collection", "outputs": [{"name": "output", "type": "input"}], "position": {"bottom": 322.5625, "height": 137, "left": 40.0625, "right": 240.0625, "top": 185.5625, "width": 200, "x": 40.0625, "y": 185.5625}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0", "tool_shed_repository": {"changeset_revision": "90981f86000f", "name": "collapse_collections", "owner": "nml", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"filename\": {\"add_name\": \"false\", \"__current_case__\": 1}, \"input_list\": {\"__class__\": \"ConnectedValue\"}, \"one_header\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "5.1.0", "type": "tool", "uuid": "a6b27681-0df4-402d-8381-1171dc02a4ae", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "f26e34eb-e8c8-4980-a352-0a4412f6c32a"}]}, "11": {"annotation": "VCF data is converted into TAB-delimited format", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpsift/snpSift_extractFields/4.3+t.galaxy0", "errors": null, "id": 11, "input_connections": {"input": {"id": 9, "output_name": "snpeff_output"}}, "inputs": [], "label": null, "name": "SnpSift Extract Fields", "outputs": [{"name": "output", "type": "tabular"}], "position": {"bottom": 854.546875, "height": 137, "left": -27.953125, "right": 172.046875, "top": 717.546875, "width": 200, "x": -27.953125, "y": 717.546875}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/snpsift/snpSift_extractFields/4.3+t.galaxy0", "tool_shed_repository": {"changeset_revision": "5fab4f81391d", "name": "snpsift", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"empty_text\": \".\", \"extract\": \"CHROM POS FILTER REF ALT DP AF DP4 SB EFF[*].IMPACT EFF[*].FUNCLASS EFF[*].EFFECT EFF[*].GENE EFF[*].CODON EFF[*].AA EFF[*].TRID DP4[2] DP4[3]\", \"input\": {\"__class__\": \"ConnectedValue\"}, \"one_effect_per_line\": \"true\", \"separator\": \"\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "4.3+t.galaxy0", "type": "tool", "uuid": "01968899-49e4-48a2-8c1c-c6dbc9862de9", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "0798dc63-1650-4868-a02a-0e5da6930eeb"}]}, "12": {"annotation": "Building a tree from consensus \"alignments\"", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/rapidnj/rapidnj/2.3.2", "errors": null, "id": 12, "input_connections": {"alignments": {"id": 10, "output_name": "output"}}, "inputs": [], "label": "Textual representation of a tree", "name": "Join neighbors", "outputs": [{"name": "distances", "type": "nhx"}], "position": {"bottom": 530.546875, "height": 179, "left": 287.046875, "right": 487.046875, "top": 351.546875, "width": 200, "x": 287.046875, "y": 351.546875}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/rapidnj/rapidnj/2.3.2", "tool_shed_repository": {"changeset_revision": "9f4a66e22580", "name": "rapidnj", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"__job_resource\": {\"__job_resource__select\": \"no\", \"__current_case__\": 0}, \"alignment_type\": \"d\", \"alignments\": {\"__class__\": \"ConnectedValue\"}, \"bootstrap\": null, \"evolution_model\": \"kim\", \"no_negative_length\": \"false\", \"output_format\": \"t\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "2.3.2", "type": "tool", "uuid": "0c47d551-6534-43cd-967d-fe8b18593532", "workflow_outputs": [{"label": null, "output_name": "distances", "uuid": "1ee0283f-6cdb-45e6-b651-ed32d85dd28e"}]}, "13": {"annotation": "A collection of tab-delimited files is collapsed into a single Ono", "content_id": "toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0", "errors": null, "id": 13, "input_connections": {"input_list": {"id": 11, "output_name": "output"}}, "inputs": [], "label": null, "name": "Collapse Collection", "outputs": [{"name": "output", "type": "input"}], "position": {"bottom": 880.546875, "height": 137, "left": 258.0625, "right": 458.0625, "top": 743.546875, "width": 200, "x": 258.0625, "y": 743.546875}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/nml/collapse_collections/collapse_dataset/5.1.0", "tool_shed_repository": {"changeset_revision": "90981f86000f", "name": "collapse_collections", "owner": "nml", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"filename\": {\"add_name\": \"true\", \"__current_case__\": 0, \"place_name\": \"same_multiple\"}, \"input_list\": {\"__class__\": \"ConnectedValue\"}, \"one_header\": \"true\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "5.1.0", "type": "tool", "uuid": "a6979a96-8ffd-4dec-a2eb-9511e7839cc1", "workflow_outputs": [{"label": null, "output_name": "output", "uuid": "58c1e48c-cd78-47f0-89d6-58cc93daef77"}]}, "14": {"annotation": "Generating a tree display", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/newick_utils/newick_display/1.6+galaxy1", "errors": null, "id": 14, "input_connections": {"fileNewick": {"id": 12, "output_name": "distances"}}, "inputs": [{"description": "runtime parameter for tool Newick Display", "name": "fileNewick"}], "label": "Phylogenetic tree", "name": "Newick Display", "outputs": [{"name": "output", "type": "svg"}], "position": {"bottom": 312.5625, "height": 137, "left": 519.0625, "right": 719.0625, "top": 175.5625, "width": 200, "x": 519.0625, "y": 175.5625}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/newick_utils/newick_display/1.6+galaxy1", "tool_shed_repository": {"changeset_revision": "b4163d2f64ab", "name": "newick_utils", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"condbranchlength\": {\"branchlength\": \"false\", \"__current_case__\": 1}, \"condbranchsupport\": {\"branchsupport\": \"false\", \"__current_case__\": 1}, \"fileNewick\": {\"__class__\": \"RuntimeValue\"}, \"leafFont\": \"sans-serif\", \"leafSize\": \"20px\", \"legend\": \"\", \"outformat\": \"png\", \"radial\": \"true\", \"scalebar\": \"true\", \"width\": \"4000\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.6+galaxy1", "type": "tool", "uuid": "7bdda658-bbfc-4f96-ab3b-709b4e8e2697", "workflow_outputs": [{"label": "Phylogenetic tree (PDF)", "output_name": "output", "uuid": "56368230-ee08-48d0-a7c4-932781f588d9"}]}, "15": {"annotation": "We are removing the file header", "content_id": "Remove beginning1", "errors": null, "id": 15, "input_connections": {"input": {"id": 13, "output_name": "output"}}, "inputs": [], "label": null, "name": "Remove beginning", "outputs": [{"name": "out_file1", "type": "input"}], "position": {"bottom": 794.546875, "height": 116, "left": 536.046875, "right": 736.046875, "top": 678.546875, "width": 200, "x": 536.046875, "y": 678.546875}, "post_job_actions": {}, "tool_id": "Remove beginning1", "tool_state": "{\"input\": {\"__class__\": \"ConnectedValue\"}, \"num_lines\": \"1\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.0.0", "type": "tool", "uuid": "ff0ddefd-0898-451f-9dff-667efd2eb793", "workflow_outputs": [{"label": null, "output_name": "out_file1", "uuid": "3eb63d87-e365-4165-baf4-ad03aeaa952a"}]}, "16": {"annotation": "Here we are grouping by Position, Reference allele, and Alternative allele", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "errors": null, "id": 16, "input_connections": {"in_file": {"id": 15, "output_name": "out_file1"}}, "inputs": [], "label": null, "name": "Datamash", "outputs": [{"name": "out_file", "type": "tabular"}], "position": {"bottom": 924.546875, "height": 116, "left": 803.0625, "right": 1003.0625, "top": 808.546875, "width": 200, "x": 803.0625, "y": 808.546875}, "post_job_actions": {"HideDatasetActionout_file": {"action_arguments": {}, "action_type": "HideDatasetAction", "output_name": "out_file"}}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "tool_shed_repository": {"changeset_revision": "562f3c677828", "name": "datamash_ops", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"grouping\": \"3,5,6\", \"header_in\": \"true\", \"header_out\": \"false\", \"ignore_case\": \"false\", \"in_file\": {\"__class__\": \"ConnectedValue\"}, \"need_sort\": \"true\", \"operations\": [{\"__index__\": 0, \"op_name\": \"countunique\", \"op_column\": \"1\"}, {\"__index__\": 1, \"op_name\": \"unique\", \"op_column\": \"1\"}], \"print_full_line\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.1.0", "type": "tool", "uuid": "8ed33929-a75d-42fe-993a-db7be898683c", "workflow_outputs": []}, "17": {"annotation": "Here additional grouping summarizes types of changes that exist in this sample", "content_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "errors": null, "id": 17, "input_connections": {"in_file": {"id": 16, "output_name": "out_file"}}, "inputs": [{"description": "runtime parameter for tool Datamash", "name": "in_file"}], "label": "Substitution matrix", "name": "Datamash", "outputs": [{"name": "out_file", "type": "tabular"}], "position": {"bottom": 868.546875, "height": 137, "left": 1078.046875, "right": 1278.046875, "top": 731.546875, "width": 200, "x": 1078.046875, "y": 731.546875}, "post_job_actions": {}, "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/datamash_ops/datamash_ops/1.1.0", "tool_shed_repository": {"changeset_revision": "562f3c677828", "name": "datamash_ops", "owner": "iuc", "tool_shed": "toolshed.g2.bx.psu.edu"}, "tool_state": "{\"grouping\": \"2,3\", \"header_in\": \"false\", \"header_out\": \"false\", \"ignore_case\": \"false\", \"in_file\": {\"__class__\": \"RuntimeValue\"}, \"need_sort\": \"true\", \"operations\": [{\"__index__\": 0, \"op_name\": \"count\", \"op_column\": \"1\"}], \"print_full_line\": \"false\", \"__page__\": null, \"__rerun_remap_job_id__\": null}", "tool_version": "1.1.0", "type": "tool", "uuid": "d116b621-a706-4b31-ae4f-16466eeaeda1", "workflow_outputs": [{"label": "Substitution frequency matrix", "output_name": "out_file", "uuid": "b0d49893-ca7e-4779-856c-48eba92ac42d"}]}}, "tags": ["viral", "bacterial", "quickies", "variation", "phylogeny"], "uuid": "72a03e89-abd9-46d5-a7c6-b88717989e19", "version": 14}
```